packages/ckeditor5-image/tests/imageinsert/imageinsertviaurlui.js

```diff
@@ -186,6 +191,7 @@ describe( 'ImageInsertViaUrlUI', () => {
 
 		testSubmit( 'accept button', () => acceptButton.fire( 'execute' ) );
 
+		// Can not be checked by simulating enter click because of unit test limitations
```

Comment:
It's a good approach to describe "not so obvious" code, however this could use some more details to be clear. You could describe that we fire form `submit` event to simulate what browser does on `enter` press instead of trying to trigger `enter` key event on a button directly.